code/index.html

```diff
@@ -4,9 +4,72 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@400;500;600&display=swap" rel="stylesheet">
     <title>Project 4: Pizzabot</title>
   </head>
   <body>
-    <h1>This is your Pizza Bot</h1>
+    <main>
+      <section>
+        <div class="welcome">
+          <h1>Claire & Claudia's pizza ordering service &#127829</h1>
+        </div>
+
+        <!-- Greeting welcoming the customer -->
+        <div class="cards">
+          <h3 id="greeting"></h3>
+          <h3 id="menu" class="menu"><li></li></h3>
+        </div>
+
+        <!-- First form section where user will input username & name of pizza -->
+        <div class="cards">
+          <h3> &#129302 Before we get started with your order please enter your username:</h3> 
+          <form action="#" autocomplete="off" onsubmit="enteredPizzaName(event)" method="POST" id="formOne" class="form-1">
+            <!-- Enter username -->
+            <label for="userName">
+              <input type="text" id="userName" name="userName" placeholder="Username" required>
+            </label>
+
+            <!-- Enter the name of the pizza -->
+            <h3 id=choosePizza></h3> 
+            <label for="enterPizza">
+              <input type="text" id="enterPizza" name="enterPizza" placeholder="Type in pizza flavour" required></select>
+            </label>
+              <input type="submit" name="submit" value="Enter" class="submit-button">
```

Comment:
Clicking this will trigger the event defined in the `onsubmit` attribute in the form.